Agreements/Internal/Trent_Wal.txt

```diff
@@ -0,0 +1,2 @@
+COPX:*:*:NANTI:EGGP:*:Trent:Wallasey:*:10000:NANTI
+COPX:*:*:NANTI:EGNR:*:Trent:Wallasey:*:10000:NANTI
```

Comment:
Trent or Stafa?

Reply:
On STAR routing LESTA-TNT-NANTI...traffic handed from AC Dty FL200 Elvos to PC TRENT and FL100 to PC WAL and then to GPAPP

Reply:
Think it's actually Stafa to Wallasey. Daventry hands to Trent all MTMA traffic 200 lvl ELVOS, then no standing agreement between Trent and Stafa, but Stafa hands to Wallasey.

![image](https://user-images.githubusercontent.com/12017284/62325728-39de0a80-b4a4-11e9-8281-c8901273c27f.png)

Reply:
This one’s still reading Trent to me

Reply:
Sorry, did not see this comment. 

Now fixed.